docs/en/02_Developer_Guides/14_Files/01_Image.md

```diff
@@ -18,60 +18,81 @@ Allows upload of images through limiting file extensions with `setAllowedExtensi
+in this way are cached for later use. In most cases the pixel aspect ratios of
+images are preserved (meaning images are not stretched).
+
+Here are some examples, assuming the `$Image` object has dimensions of 200x100px:
 
 	:::php
-	// manipulation functions
-	$image->resize(width,height); // Basic resize, just skews the image
-	$image->resizeRatio(width,height) // Resizes an image with max width and height
-	$image->paddedResize(width,height) // Adds padding after resizing to width or height.
-	$image->croppedImage(width,height) // Crops the image from the centre, to given values.
-	$image->resizeByHeight(height) // Maximum height the image resizes to, keeps proportion
-	$image->resizeByWidth(width) // Maximum width the image resizes to, keeps proportion 
-	$image->greyscale(r,g,b) // alters image channels ===
+	// Scaling functions
+	$Image.ScaleWidth(150) // Returns a 150x75px image
+	$Image.ScaleMaxWidth(100) // Returns a 100x50px image (like ScaleWidth but prevents up-sampling)
+	$Image.ScaleHeight(150) // Returns a 300x150px image (up-sampled. Try to avoid doing this)
+	$Image.ScaleMaxHeight(150) // Returns a 200x100px image (like ScaleHeight but prevents up-sampling)
+	$Image.Fit(300,300) // Returns an image that fits within a 300x300px boundary, resulting in a 300x150px image (up-sampled)
+	$Image.FitMax(300,300) // Returns a 200x100px image (like Fit but prevents up-sampling)
+	
+	// Cropping functions
+	$Image.Fill(150,150) // Returns a 150x150px image resized and cropped to fill specified dimensions (up-sampled)
+	$Image.FillMax(150,150) // Returns a 100x100px image (like Fill but prevents up-sampling)
+	$Image.ClipWidth(150) // Returns a 150x100px image (trims excess pixels off the x axis from the center)
+	$Image.ClipHeight(50) // Returns a 200x50px image (trims excess pixels off the y axis from the center)
 	
-	// values
-	$image->getHeight() // Returns the height of the image.
-	$image->getWidth() // Returns the width of the image
-	$image->getOrienation() // Returns a class constant: ORIENTATION_SQUARE or ORIENTATION_PORTRAIT or ORIENTATION_LANDSCAPE
+	// Padding functions (add space around an image)
+	$Image.SetSize(100,100) // Returns a 100x100px padded image, with white bars added at the top and bottom
+	$Image.Pad(100, 100, CCCCCC) // Same as SetSize but you can choose the background colour (grey in this example)
+	
+	// Metadata
+	$Image.Width // Returns width of image
+	$Image.Height // Returns height of image
+	$Image.Orientation // Returns Orientation
+	$Image.Title // Returns the friendly file name
+	$Image.Name // Returns the actual file name
+	$Image.FileName // Returns the actual file name including directory path from web root
+	$Image.Link // Returns relative URL path to image
+	$Image.AbsoluteLink // Returns absolute URL path to image
+
+Image methods are chainable. Example: `<body style="background-image:url($Image.ScaleWidth(800).ClipHeight(800).Link)">`
```

Comment:
Just a question; Have you checked that the generated URL for these chained methods contains the operations and doesn't just cache on whatever the last operation was? Eg.. `_resampled/setwidth400_setheight200_FileName.jpg`